libs/core/src/components/pds-button/pds-button.scss

```diff
@@ -1,140 +1,151 @@
-  --border-color-disabled-secondary: var(--pine-color-neutral-grey-300);
-  --color-secondary: var(--pine-color-neutral-charcoal-400);
-  --color-disabled-secondary: var(--pine-color-neutral-charcoal-100);
-  --outline-color-secondary: var(--pine-color-primary-200);
+  --color-border-secondary-default: var(--pine-color-grey-400);
+  --color-border-secondary-disabled: var(--pine-color-grey-300);
+  --color-border-secondary-focus: var(--pine-color-grey-300);
+  --color-border-secondary-hover: var(--pine-color-grey-500);
+  --color-text-secondary-default: var(--pine-color-text-body-default);
+  --color-text-secondary-disabled: var(--pine-color-text-action-primary-disabled);
+  --color-outline-secondary: var(--pine-color-blue-200);
 
   // accent
-  --background-color-accent: var(--pine-color-primary-300);
-  --background-color-hover-accent: var(--pine-color-primary-400);
-  --background-color-disabled-accent: var(--pine-color-primary-100);
-  --color-accent: var(--pine-color-base-white);
-  --color-disabled-accent: var(--pine-color-primary-200);
-  --outline-color-accent: var(--pine-color-primary-200);
+  --color-background-accent-default: var(--pine-color-background-action-accent-default);
+  --color-background-accent-disabled: var(--pine-color-background-action-accent-disabled);
+  --color-background-accent-hover: var(--pine-color-background-action-accent-hover);
+  --color-text-accent-default: var(--pine-color-text-action-accent-default);
+  --color-text-accent-disabled: var(--pine-color-text-action-accent-disabled);
+  --color-outline-accent: var(--pine-color-blue-200);
 
   // destructive
-  --background-color-destructive: var(--pine-color-red-300);
-  --background-color-hover-destructive: var(--pine-color-red-400);
-  --background-color-disabled-destructive: var(--pine-color-red-100);
-  --color-destructive: var(--pine-color-base-white);
-  --color-disabled-destructive: var(--pine-color-red-200);
-  --outline-color-destructive: var(--pine-color-red-200);
+  --color-background-destructive-default: var(--pine-color-background-action-destructive-default);
+  --color-background-destructive-disabled: var(--pine-color-background-action-destructive-disabled);
+  --color-background-destructive-hover: var(--pine-color-background-action-destructive-hover);
+  --color-text-destructive-default: var(--pine-color-text-action-destructive-default);
+  --color-text-destructive-disabled: var(--pine-color-text-action-destructive-disabled);
+  --color-outline-destructive: var(--pine-color-red-200);
+
+  --spacing-disclosure-icon-margin-inline-end: var(--pine-spacing-0);
+  --spacing-disclosure-margin-inline-start: var(--pine-spacing-100);
+  --spacing-icon-margin-inline-end: var(--pine-spacing-100);
+  --spacing-padding-block: var(--pine-spacing-100);
+  --spacing-padding-inline: var(--pine-spacing-200);
+  --spacing-unstyled-padding: var(--pine-spacing-0);
+  --spacing-unstyled-margin: var(--pine-spacing-0);
```

Comment:
I may be misunderstanding the spacing naming, but wouldn't we want to try and prevent duplicates?

Reply:
@pixelflips I understand your point. It feels like a case by case basis for me. I'm thinking in some cases repetition may be fine if it makes sense in the context it's being used. What do you think?

Reply:
That makes sense, but those to me would be the outliers. For instance the `--spacing-unstyled-margin` and `--spacing-unstyled-padding` could be combined into something like `--spacing-unstyled-default` and then if they ever have different values we could separate them. That just seems more DRY to me and is one less variable to maintain over time as an example. 

Definitely not a blocker this time around.

Reply:
I like this explanation. Thanks for this. Just updated the tokens.